ibis/expr/operations/relations.py

```diff
@@ -408,29 +408,40 @@ class SQLQueryResult(Relation):
 
 
 @public
-class SQLStringView(PhysicalTable):
-    """A view created from a SQL string."""
+class View(PhysicalTable):
+    """A view created from an expression."""
 
     child: Relation
-    query: str
 
     @attribute
     def schema(self):
-        # TODO(kszucs): avoid converting to expression
-        backend = self.child.to_expr()._find_backend()
-        return backend._get_schema_using_query(self.query)
+        return self.child.schema
 
 
 @public
-class View(PhysicalTable):
-    """A view created from an expression."""
+class SQLStringView(Relation):
+    """A view created from a SQL string."""
 
-    child: Relation
-    name: str
+    child: View
+    query: str
+
+    values = FrozenDict()
 
     @attribute
     def schema(self):
-        return self.child.schema
+        import sqlglot as sg
+
+        # TODO(kszucs): avoid converting to expression
+        op = self.child
+        child = op.to_expr()
+        backend = child._find_backend()
+
+        obj = backend._to_sqlglot(child)
```

Comment:
Could you please move the sqlglot related code to `_get_schema_using_query()`? Either passing child as an optional argument to it, or if `_get_schema_using_query` is only used for this purpose then it could be even renamed to reflect this is for querying the sqlstringview schema.

Reply:
Yep, I'll look into it.

Reply:
Moved this functionality to the base sqlglot backend.